specs/signature-specification.md

```diff
@@ -127,43 +84,44 @@ A signature envelope consists of the following components:
 
 A signature envelope is `e = {m, v, u, s}` where `s` is signature.
 
-This specification defines the set of signed and unsigned attributes that make up a valid The Notary v2 signature. This specification aims to be be agnostic of signature envelope format (e.g. COSE, JWS), details of encoding the envelope in a specific signature envelope format are covered in in separate specs.
+This specification defines the set of signed and unsigned attributes that make up a valid Notary Project signature. This specification aims to be be agnostic of signature envelope format (e.g. COSE, JWS), details of encoding the envelope in a specific signature envelope format are covered in in separate specs.
 
-Notary v2 supports the following envelope formats:
+The Notary Project signature supports the following envelope formats:
 
 - [JWS](./signature-envelope-jws.md)
-- [COSE Sign1](./signature-envelope-cose.md)
+- [COSE](./signature-envelope-cose.md)
 
 ### Payload
 
-Notary v2 payload is a JSON document with media type `application/vnd.cncf.notary.payload.v1+json` and has following properties.
+The Notary Project signature payload is a JSON document with media type `application/vnd.cncf.notary.payload.v1+json` and has following properties.
 
 - `targetArtifact` : Required property whose value is the descriptor of the target artifact manifest that is being signed. Only [OCI descriptor][oci-descriptor] is supported.
-  - Descriptor MUST contain `mediaType`, `digest`, `size` fields.
-  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules]. Notary v2 uses annotations for storing both Notary specific and user defined metadata. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary v2 and MUST NOT be used outside this specification.
-  - Descriptor MAY contain `artifactType` field for artifact manifests, or the `config.mediaType` for `oci.image` based manifests.
+  - Descriptor MUST contain `mediaType`, `digest`, and `size` fields.
+  - Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules][annotation-rules]. The Notary Project signature uses annotations for storing both Notary Project specific and user defined metadata. The prefix `io.cncf.notary` in annotation keys is reserved for use in Notary Project signature and MUST NOT be used outside this specification.
+  - Descriptor MAY contain `artifactType` field for the `config.mediaType` of OCI image manifests.
```

Comment:
I'm not sure if this is correct. In notation-go, we are sanitizing the target artifact to just `mediaType`, `digest`, `size`, and `annotations`.
Looks like `Descriptor MAY contain artifactType field for artifact manifests` was for `artifact manifest` only? We are using image manifest now.

Reply:
This description was not added for OCI image spec v1.0.0-rc4. It was in the original version, that is, `artifactType` field MAY be used in `Descriptor` from specification point of view. From the implementation points of view, we don't use it this way today.